packages/blade/src/components/Link/_decisions/decisions.md

```diff
@@ -0,0 +1,62 @@
+# Link Decisions <!-- omit in toc -->
+- [Design](#design)
+- [Internal BaseLink Component](#internal-baselink-component)
+  - [BaseLink API](#baselink-api)
+- [Link Component](#link-component)
+  - [Link API](#link-api)
+- [Component Breakdown](#component-breakdown)
+- [Accessibility](#accessibility)
+- [Open Questions](#open-questions)
+
+## Design
+[Figma Link](https://www.figma.com/file/jubmQL9Z8V7881ayUD95ps/Blade---Payment-Light?node-id=12699%3A147155) to the **Link** component
+
+
+## Internal BaseLink Component
+Internal component that exposes certain extra props to enable creation of Component Level Action Link like `positive`, `negative`, `neutral`, `information`, `notice`
+
+### BaseLink API
+| Prop | Type | Required | Default | Description |
+|---|---|---|---|---|
+| variant | `anchor`, `button` | *No* | `anchor` | The variant of the Link to be rendered.<br><br>Note: `button` will look like an inline Link component (no spacing; can be rendered alongside other text) but will be rendered as a `button` element. |
+| intent |  `positive`,`negative`, `notice`, `information`, `neutral` | *No* | undefined | The intent of the link to be rendered.<br><br>  Note: <br> - We will use this to set the intent for internal Blade components that have positive, negative, notice, info, or neutral link buttons like an Alert component. |
```

Comment:
How would default `undefined` intent translate design wise 🤔

Reply:
When the intent is not defined, it'll behave like a normal link (blue in color) but when there is an intent, it will look as per the intent (red for negative, yellow for notice, etc.)